src/generic/grid.cpp

```diff
@@ -9117,26 +9161,28 @@ void wxGrid::SetCornerLabelTextOrientation( int textOrientation )
 
 void wxGrid::SetRowLabelValue( int row, const wxString& s )
 {
-    if ( m_table )
+    if ( m_table && m_rowLabelWidth > 0 )
     {
         m_table->SetRowLabelValue( row, s );
         if ( ShouldRefresh() )
```

Comment:
OTOH it would be fine to add a check for row labels being shown (BTW, if you wish to add `bool AreRowLabelsShown()` instead of comparing `m_rowLabelWidth` with 0, please do it) here.